### DISCUSSION TITLE
Center Aligning Mermaid Graphs

### DISCUSSION THREAD

**Discussion by ghost** [imfing/hextra#17] [Q&A] [answered]
[This page](https://imfing.github.io/hextra/docs/guide/diagrams/#mermaid) shows the graph aligned to the left. Is there any way I can align it to the center?

**Comment by imfing** [comment, marked as answer]
@saurabh-mish it's possible by adding a `<div align="center">` around the diagram:

````
<div align="center">

```mermaid
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```

</div>
````

Preview:

<img width="1038" alt="image" src="https://github.com/imfing/hextra/assets/5097752/2fc3ae8b-6344-433d-8fdb-320206669351">

**Reply by ghost** [reply]
Thank you !
